packages/maskbook/src/extension/background-script/EthereumServices/providers/Maskbook.ts

```diff
@@ -54,6 +54,7 @@ export function createWeb3({
     privKeys?: string[]
 } = {}) {
     url = url || getChainRPC(chainId, SEED)
+    if (!url) throw new Error(`Unknown chain id: ${chainId}`)
```

Comment:
```suggestion
    // TODO: Check chainId in `getChainRPC`, and throw this error from it in case.
    if (!url) throw new Error(`Unknown chain id: ${chainId}`)
```
Should this error be threw from `getChainRPC`? Maybe refactor in someday?

Reply:
I found that `getChainRPC` will throw an error for an unknown chain id.